packages/stack/src/views/Stack/Card.tsx

```diff
@@ -271,6 +271,14 @@ export default class Card extends React.Component<Props> {
           velocity = nativeEvent.velocityX;
         }
 
+        if (
+          gestureDirection === 'horizontal-inverted' ||
+          gestureDirection === 'vertical-inverted'
+        ) {
+          translation = Math.abs(translation);
+          velocity = Math.abs(translation);
```

Comment:
I think you meant `velocity = Math.abs(velocity);`?

Reply:
Yes, my bad, sorry.